quirks/password-rules.json

```diff
@@ -162,6 +162,9 @@
     "propelfuels.com": {
         "password-rules": "minlength: 6; maxlength: 16;"
     },
+    "reddit.com": {
+        "password-rules": "minlength: 6;"
```

Comment:
Given that no password manager I’m aware of would attempt to, by default, generate a password with fewer than 6 characters, I’m not sure this is worth tracking in the project. Like, I don’t think it’ll have a meaningful effect.